airflow/providers/common/sql/hooks/sql.py

```diff
@@ -175,41 +207,26 @@ def get_pandas_df_by_chunks(self, sql, parameters=None, *, chunksize, **kwargs):
             yield from psql.read_sql(sql, con=conn, params=parameters, chunksize=chunksize, **kwargs)
 
     def get_records(
-        self,
-        sql: str | list[str],
-        parameters: Iterable | Mapping | None = None,
-        **kwargs: dict,
-    ):
+        self, sql: str | list[str], parameters: Iterable | Mapping | None = None
+    ) -> Any | list[Any]:
```

Comment:
Isn't `Any | list[Any] == Any`  ?

Reply:
I think that should also remove the # type: ignore[override] in subcleases?

Reply:
I changed back from `sql: str = ""` to `sql: str | list[str] = ""`.
It seems strange but without it, I can't remove `# type: ignore[override]`